js/sales.js

```diff
@@ -24,56 +24,119 @@
-            } else if(militaryHour === 24){
-                newHourItem.textContent = '12am: ' + hourSoldCookies + ' cookies';
+                newHourHead.textContent = militaryHour + ':00am';
             } else {
                 //1pm-11pm
-                newHourItem.textContent = (militaryHour - standardTimeOffSet) + 'pm: ' + hourSoldCookies + ' cookies';
+                newHourHead.textContent = (militaryHour - standardTimeOffSet) + ':00pm';
+            }
+            tableRow.appendChild(newHourHead);
+        }
+        return tableRow;
+    }
+
+    CookieStore.prototype.render = function(){
+        var tableStoreRow = document.createElement('tr');
+        tableStoreRow.setAttribute('scope', 'row');
+        var tableStoreHead = document.createElement('th');
+        tableStoreHead.textContent = this.name;
+        document.querySelector('tbody').appendChild(tableStoreRow);
+        tableStoreRow.appendChild(tableStoreHead);
+
+        calculateSimulatedCookiesPerHour(this);
+        
+        for(var i = 0; i < this.simulatedCookiesPerHour.length; i++){
+            var tableStoreData = document.createElement('td');
+            tableStoreData.textContent = this.simulatedCookiesPerHour[i];
+            tableStoreRow.appendChild(tableStoreData);
+        }
+        return tableStoreRow;
+    }
+
+    function createTotalsFooterRow(){
+        var totalsRow = document.createElement('tr');
+        totalsRow.setAttribute('scope', 'row');
+        var totalsHead = document.createElement('th');
+        totalsHead.textContent = 'Totals';
+        document.querySelector('tfoot').appendChild(totalsRow);
+        totalsRow.appendChild(totalsHead);
+
+        for(var i = 0; i < cookieStoreLocations[0].simulatedCookiesPerHour.length; i++){
+            var total = 0;
+            for(var j = 0; j < cookieStoreLocations.length; j++){
+                total += cookieStoreLocations[j].simulatedCookiesPerHour[i];
             }
-            list.append(newHourItem);
+            var tableTotalData = document.createElement('td');
+            tableTotalData.textContent = total;
+            totalsRow.appendChild(tableTotalData);
+        }
+        return totalsRow;
+    }
+
+    function createStoreTotals(location, tableRow){
+        var total = 0;
+        for(var i = 0; i < location.simulatedCookiesPerHour.length; i++){
+            total += location.simulatedCookiesPerHour[i];
         }
+        var tableStoreTotalData = document.createElement('td');
+        tableStoreTotalData.textContent = total;
+        tableRow.appendChild(tableStoreTotalData);
+        return total;
+    }
+
+    function createDailyLocationTotalHeader(tableRow){
+        var dailyLocationTotal = document.createElement('th');
+        dailyLocationTotal.setAttribute('scope', 'col');
+        dailyLocationTotal.textContent = 'Daily Location Total';
+        tableRow.appendChild(dailyLocationTotal);
     }
```

Comment:
Perhaps we could refactor this function a little to use for both the top-left cell and the top-right cell of the table? For both, we're creating a `th`, setting its scope to `col`, and appending it to the `tableRow`, so the only difference is the `textContent` - which we could make a parameter. We could also make this code a little more functional by having it return the `th` element it creates instead of mutating a `tableRow` passed in.

So then, on line 56, we could do something like, `tableRow.appendChild(createHeaderCell('Store Locations'))` and on line 35, we could do `hoursTableRow.appendChild(createHeaderCell('Daily Location Total'))`.